xml/Microsoft.VisualC.StlClr/BinaryDelegate`3.xml

```diff
@@ -36,12 +36,12 @@
     <ReturnType>TResult</ReturnType>
   </ReturnValue>
   <Docs>
-    <typeparam name="TArg1">The first delegate argument.</typeparam>
-    <typeparam name="TArg2">The second delegate argument.</typeparam>
+    <typeparam name="TArg1">The type of the first delegate argument.</typeparam>
+    <typeparam name="TArg2">The type of the second delegate argument.</typeparam>
     <typeparam name="TResult">The return type of the delegate.</typeparam>
     <param name="A_0">The first argument.</param>
     <param name="">The first argument.</param>
-    <param name="">To be added.</param>
+    <param name="">The second argument.</param>
```

Comment:
The parameters don't currently show at all:

https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualc.stlclr.binarydelegate-3?view=netframework-4.8

Reply:
yep it's a known bug :-(  we have a tracking issue for taht

but I want to add the data here when we fix the bug (and also for our compliance reports completion)